cmd/podman/kube/play.go

```diff
@@ -246,18 +246,17 @@ func play(cmd *cobra.Command, args []string) error {
 	}
 
 	for _, annotation := range playOptions.annotations {
-		splitN := strings.SplitN(annotation, "=", 2)
-		if len(splitN) != 2 {
-			return fmt.Errorf("annotation %q must include an '=' sign", annotation)
+		key, annotation, hasAnnotation := strings.Cut(annotation, "=")
```

Comment:
I realize that this usage predates your PR, but DAE find the overloading of `annotation` unreadable? Might it make sense to use the (clean, standard) `key, val` syntax instead?

Reply:
+1 from me

Reply:
Changed to `key, val, hasVal`